kfdefs/overlays/moc/smaug/opf-dashboard/kfdef.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
```

Comment:
can you add the overlays here? so it would look like: 

```
  - kustomizeConfig:
      overlays:
        - custom-image
        - cluster-role
      repoRef:
        name: manifests
        path: odh-dashboard
    name: odh-dashboard
```